src/assets/scss/6-components/molecules/_drag-and-drop-list-box.scss

```diff
@@ -0,0 +1,41 @@
+.c-drag-and-drop-list-box {
+    &.-dragging-container {
+        .c-button {
+            display: none !important;

```

Comment:
Is this `!important` necessary?

Reply:
ah, no. good catch. This rule isn't even needed at all. This was something from a client project.